github/git_refs_test.go

```diff
@@ -14,7 +14,7 @@ import (
 	"testing"
 )
 
-func TestGitService_GetRef(t *testing.T) {
+func TestGitService_GetRef_SingleResult(t *testing.T) {
```

Comment:
The naming convention we apply for tests (it's based on the convention examples) is to begin the test label – "SingleResult" in this case – with lower case.

For example, see https://github.com/google/go-github/blob/e96a1efec3efbfec63cf32984b21bf98933042a2/github/pulls_comments_test.go#L18, https://github.com/google/go-github/blob/e96a1efec3efbfec63cf32984b21bf98933042a2/github/pulls_comments_test.go#L51.

Source: https://godoc.org/testing#hdr-Examples.

So, this should be renamed to `TestGitService_GetRef_singleResult`.

However, let's be more specific and say it's single ref, rather than single result.

So, `TestGitService_GetRef_singleRef`.